target/linux/ath79/dts/qca9531_kuwfi_830d.dts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9531_ap147_010_reference_board.dtsi"
+
+/ {
+	model = "KuWfi 830D";
+	compatible = "kuwfi,830d", "qca,qca9531";
```

Comment:
I'd put model after compatible (although this might just be common use).

Reply:
fixed